### PR TITLE
Debezium SQL Server Connector native support #1193

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Issue:https://github.com/apache/camel-quarkus/issues/1193

[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

File: extensions/debezium-sqlserver/runtime/src/main/doc/usage.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+=== Encryption
+
+SQL server may require encryption. In that case `quarkus.ssl.native=false` parameter has to be added during execution.
+See https://quarkus.io/guides/native-and-ssl for more details.
+
+=== Integration testing
```
Comment (jamesnetherton) [inline extensions/debezium-sqlserver/runtime/src/main/doc/usage.adoc:6]: Maybe move this section into a dedicated README in the debezium itest module. It's not really required information for the most users.

File: integration-tests/debezium/pom.xml [hunk 2]
```diff
@@ -39,8 +40,11 @@
         <!--     mvn process-resources -Pformat    from the root directory -->
         <mvnd.builder.rule>camel-quarkus-debezium-postgres-deployment,camel-quarkus-debezium-mysql-deployment,camel-quarkus-support-policy-deployment</mvnd.builder.rule>
```
Comment (jamesnetherton) [inline integration-tests/debezium/pom.xml:41]: `camel-quarkus-debezium-sqlserver-deployment` seems to be missing.

Comment (JiriOndrusek) [reply integration-tests/debezium/pom.xml]: @jamesnetherton Both changes are added to the PR.

**Comment by jamesnetherton** [comment]
@JiriOndrusek please can you rebase and resolve conflicts.

**Comment by JiriOndrusek** [comment]
@jamesnetherton rebased and conflicts resolved

**Pull request merged**